src/components/utils.css

```diff
@@ -9,8 +9,65 @@
 }
 .component-container {
   padding: 1rem;
-  margin: auto;
+  margin: 1rem auto;
```

Comment:
remove margins from right and left than your component container would be aligned to left

Reply:
this was a deliberate choice